docs/system/system-metrics.md

```diff
@@ -73,6 +75,31 @@ Resource attributes related to a host, SHOULD be reported under the `host.*` nam
 > * SHOULD introduce a control mechanism to allow users to opt-in to the new
 >   conventions once the migration plan is finalized.
 
+## General Metrics
+
+### Metric: `system.uptime`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.system.uptime -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `system.uptime` | Counter | `s` | The time the system has been running [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
I believe this is stated elsewhere, but this should be a gauge.

Copying rationale from the uptime monitoring OTEP:

> uptime is reported as a gauge with the value of the number of seconds that the process has been up. This is written as a gauge because users want to know the actual value of the number of seconds since the last restart to satisfy the use cases above. Sums are not a good fit for these use cases because most metric backends tend to default cumulative monotonic sums to rate-calculations, and have overflow handling that is undesired for this use case.
>
> Sums report a total value that has accumulated over a time window; it is valid, for instance, to subtract the current value of a cumulative sum and restart the start timestamp to now. (OpenTelemetry's Prometheus receiver does this, for instance.)
An intended use case of a sum is to produce a meaningful value when aggregating away labels using sum. Such aggregations are not meaningful in the above use cases.

Reply:
updated